### ISSUE TITLE
Dropdown element first quetion answer equal to 0 not possible 

### ISSUE THREAD

**Issue by tolgakutlu** [awsmug/torro-forms#439]
On my website https://www.dutchyipsstudy.nl/start-dys/ I have 3 dropdown elements (questions 15, 16 and 17) where you can choice as an answer 0. But there is a bug. Because if I choice 0 and sent the form the system says you must enter something there. Try it choice on questions 15, 16 and 17 as a answer 15 0 16 0 17 0 and sent it you wil get a error and the values will be empty on that questions if the value is equel to 0. But if you choice 1 0 0 as answer there will not a error but if you forget a required question the answers with a value 0 will be empty and you must choice is again.

**Comment by mahype** [comment]
Is the survey made with version 1.0.0?

**Comment by tolgakutlu** [comment]
@mahype yes it is. The problem is this the first dropdown element with a answer as equal to 0 you can't choose 0. I upgraded today to 1.0.0 the problem is still in place

**Comment by mahype** [comment]
I am sorry, but I could not reproduce that. But there was a problem on all zero values which I fixed by commit 97dfa2a. I think there is a good chance that this problem is also solved by this bugfix. ;)

https://github.com/awsmug/torro-forms/archive/1.0.1.zip
